sherpa/csrc/sherpa_online.cc

```diff
@@ -63,46 +63,71 @@ Note: You can get pre-trained models for testing by visiting
  * @return Return the decoded results.
  */
 static std::vector<std::string> DecodeWaves(
-    sherpa::OnlineAsr &online_asr,  // NOLINT
```

Comment:
Could you also fix
https://github.com/k2-fsa/sherpa/blob/21f829ce5816ce1ef1b00c7e7f194b69386f1dca/sherpa/csrc/sherpa_online.cc#L54